tests/api/v1/subjects/get.js

```diff
@@ -19,6 +19,7 @@ const u = require('./utils');
 const Subject = tu.db.Subject;
 const path = '/v1/subjects';
 const expect = require('chai').expect;
+const filterSubjByTags = require('../../../../config').filterSubjByTags;
```

Comment:
will always be false unless we explicitly set env var when we call the test (see examples in package.json)